packages/types/src/lib/interfaces/session-sigs.ts

```diff
@@ -0,0 +1,23 @@
+import { AuthSig } from '../interfaces';
+
+export interface ParsedSignedMessage {
+  // Known keys
+  URI?: string;
+  Version?: string;
+  'Chain ID'?: string;
+  Nonce?: string;
+  'Issued At'?: string;
+  'Expiration Time'?: string;
+  Resources?: string[]; // Should be an array of strings
+  // Dynamic keys
+
+  [key: string]: any;
```

Comment:
nit: if we are not going to operate over these it is better to use `unknown`
```suggestion
  [key: string]: unknown;
```

But there is an `expiration` that makes TS not like this definition